model_compression_toolkit/quantizers_infrastructure/pytorch/inferable_quantizers/activation_inferable_quantizers/activation_symmetric_inferable_quantizer.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+
+from model_compression_toolkit.core.common.constants import FOUND_TORCH
+from model_compression_toolkit.quantizers_infrastructure.common.base_inferable_quantizer import QuantizationTarget
+
+
+if FOUND_TORCH:
+    import torch
+    from model_compression_toolkit.quantizers_infrastructure.pytorch.quantizer_utils import \
+        get_activation_symmetric_quantization_range_and_scale
+    from model_compression_toolkit.quantizers_infrastructure.pytorch.inferable_quantizers \
+        .base_symmetric_inferable_quantizer import \
+        BaseSymmetricInferableQuantizer
+
+
+    class ActivationSymmetricInferableQuantizer(BaseSymmetricInferableQuantizer):
```

Comment:
A question regarding all quantizers - why aren't we using the same quantization functions that we have in "core"? i.e., here, why not use [this](https://github.com/sony/model_optimization/blob/f0186fa9d5583a0631596bc70d37234eb8d2ace2/model_compression_toolkit/core/pytorch/quantizer/fake_quant_builder.py#L76)?

If this is because they quantize differently, then maybe our core quantizers have issues that we need to fix?

Reply:
Because it won't be in the MCT in the future